admin/tools/docker/configured/scripts/wait.sh

```diff
@@ -20,14 +20,19 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 
+# Docker utility:
+# return when all Qserv services are up and running
 
-# Stop Qserv on current node
-
-# @author  Fabrice Jammes, IN2P3/SLAC
+# @author  Fabrice JAMMES, IN2P3
 
 set -e
 
-DIR=$(cd "$(dirname "$0")"; pwd -P)
-. $DIR/params.sh
+QSERV_RUN_DIR=/qserv/run
+
+
+while ! "$QSERV_RUN_DIR"/bin/qserv-status.sh > /dev/null
+    do
+    sleep 1
```

Comment:
A comment is in shmux/run-multinode-tests.sh where I call this script, I add also one here now.